vector/src/main/res/values/config.xml

```diff
@@ -21,7 +22,7 @@
     <string name="pusher_app_id" translatable="false">im.vector.app.android</string>
 
     <!-- preferred jitsi domain -->
-    <string name="preferred_jitsi_domain" translatable="false">jitsi.riot.im</string>
+    <string name="preferred_jitsi_domain" translatable="false">meet.element.io</string>
```

Comment:
out of interest, is there a ticket with this request? wondering if there's some other tickets we can link to help leave a paper trail

Reply:
I do not think so. I could have created one, or maybe the people working on the Jitsi configuration could have created issues on each Element clients...

Related PR on EleWeb (I didn't find any issue either):
- https://github.com/vector-im/element-web/pull/20487
- https://github.com/matrix-org/matrix-react-sdk/pull/7514